packages/docs/src/helpers/SortObjectByPropNamesHelper.ts

```diff
@@ -0,0 +1,9 @@
+export const sortObjectByPropNames = (obj: any) => {
+  const arrFromObj = Object.entries(obj);
```

Comment:
Consider using Object.keys, if it's possible. Should be though. We sorting objects by keys, so we shouldn't bother with values at all.